docs/api.rst

```diff
@@ -691,7 +691,7 @@ to handle it, which defaults to print a traceback and ignoring the exception.
     This requires :attr:`Intents.guilds` to be enabled.
 
     :param channel: The guild channel that had its pins updated.
-    :type channel: Union[:class:`abc.GuildChannel`, :class:`Thread`]
+    :type channel: Union[:class:`nextcord.TextChannel`, :class:`Thread`]
```

Comment:
this would show as `Union[nextcord.TextChannel, Thread]` so either use `:class:`TextChannel` or :class:`~nextcord.TextChannel`